scripts/Tools/getTiming

```diff
@@ -1,669 +1,620 @@
+
+        minval, maxval, found = self.gettime(label)
+        if div >= 1.0:
+            mind = minval/div
+            maxd = maxval/div
+        else:
+            mind = minval
+            maxd = maxval
+
+        pstrlen = 25
+        if mind >= 0 and maxd >= 0 and found:
+            if coff >= 0:
+                zoff = pstrlen + coff + int((datalen-clen)/2)
+                csp = offset - coff - int((datalen-clen)/2)
+                self.write(" {label:<{width1}}{cstr:<{width2}} "
+                           "{minv:8.3f}:{maxv:8.3f} \n".format(label=label,
+                                                               width1=zoff,
+                                                               cstr=cstr,
+                                                               width2=csp,
+                                                               minv=mind,
+                                                               maxv=maxd))
+            else:
+                zoff = pstrlen + offset
+                self.write(" {label:<{width1}} {minv:8.3f}:{maxv:8.3f} \n"
+                           .format(label=label, width1=zoff,
+                                   minv=mind, maxv=maxd))
+
+    def gettime2(self, heading_padded):
+        nprocs = 0
+        ncount = 0
+
+        heading = heading_padded.strip()
+        for line in self.finlines:
+            m = re.match(r'\s*%s\s*(\d+)\s*\d+\s*(\S+)'%heading, line)
+            if m:
+                nprocs = int(float(m.groups()[0]))
+                ncount = int(float(m.groups()[1]))
+                return (nprocs, ncount)
+            else:
+                m = re.match(r'\s*%s\s+(\d+)\s'%heading, line)
+                if m:
+                    nprocs = 1
+                    ncount = int(float(m.groups()[0]))
+                    return (nprocs, ncount)
+        return (0, 0)
+
+    def gettime(self, heading_padded):
+        found = False
+        heading = heading_padded.strip()
+        minval = 0
+        maxval = 0
+
+        for line in self.finlines:
+            m = re.match(r'\s*%s\s*\d+\s*\d+\s*\S+\s*\S+\s*(\d*\.\d+)'
+                         r'\s*\(.*\)\s*(\d*\.\d+)\s*\(.*\)'%heading, line)
+            if m:
+                maxval = float(m.groups()[0])
+                minval = float(m.groups()[1])
+                found = True
+                return (minval, maxval, found)
+        return (0, 0, False)
+
+
+
+    def getTiming(self):
+        atm = self.models['ATM']
+        lnd = self.models['LND']
+        rof = self.models['ROF']
+        wav = self.models['WAV']
+        ice = self.models['ICE']
+        ocn = self.models['OCN']
+        glc = self.models['GLC']
+        cpl = self.models['CPL']
+        case = Case(self.caseroot)
+        caseid = case.get_value("CASE")
+        mach = case.get_value("MACH")
+        user = case.get_value("USER")
+        continue_run = str(case.get_value("CONTINUE_RUN")).upper()
+        rundir = case.get_value("RUNDIR")
```

Comment:
On the current master these variables are cast to the correct type, no need to convert them here.